Sources/SAPCAI/UI/Common/SwiftUI/CardSectionsView.swift

```diff
@@ -133,6 +133,7 @@ extension CardSectionsView {
 
```

Comment:
@ShadowTourist  `#if DEBUG` is needed as `PreviewData` is only available in DEBUG. I pushed fix https://github.com/SAP/cloud-sdk-ios-cai/commit/ad1cdd3dedc07d2e08f504fee4bbf8a01f0a4c97 so no worries :) this is just a FYI for further use

Reply:
👍 Thanks, I check the issues, this confused me before, I thought apple have fixed preview debug model issue.
But will fail release when preview use some `Debug` data only.